operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -266,7 +270,7 @@ type KarmadaAggregatedAPIServer struct {
 	// state. Before you do it, please confirm that you understand the risks of this configuration.
 	//
 	// For supported flags, please see
-	// https://github.com/karmada-io/karmada/blob/master/cmd/aggregated-apiserver/app/options/options.go
+	// https://karmada.io/docs/reference/components/aggregated-apiserver
```

Comment:
The link is dead. should be:
https://karmada.io/docs/reference/components/karmada-aggregated-apiserver

aggregated-apiserver vs `karmada-`aggregated-apiserver

Reply:
Thanks for taking the time to review the code.

Reply:
I basically just look at the API part. :)